library/core/resource_loader/src/main/java/org/quiltmc/qsl/resource/loader/impl/ResourceLoaderImpl.java

```diff
@@ -557,17 +560,7 @@ private static List<ResourcePack> findUserStaticPacks() {
 						// that does NOT support such extended attributes (for instance, a FAT32 drive). Since these can have names of arbitrary lengths,
 						// the only way to detect them is via the shared prefix.
 						// Source: https://apple.stackexchange.com/a/14981
-						boolean fsHelper = pathAsFile.getName().startsWith("._");
-
-						if (!fsHelper) {
-							for (String fs : STATIC_USERSPACE_IGNORED_FILES) {
-								if (fs.equalsIgnoreCase(pathAsFile.getName())) {
-									fsHelper = true;
-									break;
-								}
-							}
-						}
-						if (!fsHelper) {
+						if (!pathAsFile.getName().startsWith("._") && !STATIC_USERSPACE_FILESYSTEM_HELPERS.contains(pathAsFile.getName().toLowerCase())) {
```

Comment:
More nitpicks:

I would strongly suggest not leaving this up to the system-default Locale. These are locale-insensitive Strings, so you want to use `toLowerCase(Locale.ROOT)`

Reply:
I wouldn't have thought of that, thanks!
Fixed as of latest commit.